Assets/WizardsCode/Character/Scripts/Ink/InkManager.cs

```diff
@@ -242,16 +253,19 @@ private void EraseUI()
             for (int i = 0; i < choicesPanel.transform.childCount; i++) {
                 Destroy(choicesPanel.transform.GetChild(i).gameObject);
             }
+
+            m_TextBubbleComp.ClearText();
         }
 
         private void UpdateGUI()
         {
             EraseUI();
 
             textPanel.gameObject.SetActive(true);
-            TextMeshProUGUI chunkText = Instantiate(m_StoryChunkPrefab) as TextMeshProUGUI;
-            chunkText.text = m_NewStoryText.ToString();
-            chunkText.transform.SetParent(textPanel.transform, false);
+
+            // todo - format the ink files to make the speaker name obvious
+            // todo - format the ink to make it obvious whether its dialogue
+            m_TextBubbleComp.SetText("name", m_NewStoryText.ToString(), true);

```

Comment:
in the future, we would want a non-dialogue textBubble, as well as a 'the character thought this' text bubble, and swap between them